src/main/java/uk/ac/ebi/subs/agent/services/UpdateService.java

```diff
@@ -29,6 +30,7 @@
     BsdSampleToUsiSample toUsiSample;
 
     public List<Sample> update(List<Sample> sampleList) {
+        Assert.notNull(sampleList);
```

Comment:
We are already verifying the `null` list on line 36.

Reply:
I favour throwing an error for incorrect use of the service, rather than accepting it. I can either remove line 33, or line 36. Either way, I'll make the submission service consistent with the update service.